lib/user/user-router.js

```diff
@@ -152,41 +52,8 @@ function initRouter(delegate) {
   return router;
 }
 
-function initAuthpolicyRouter(delegate) {
-  var router = express.Router();
-
-  router.route('/auth').post(function(req, res, next) {
-    var params = req && req.body && req.body.params;
-    delegate.auth(params).then(function(data) {
-      res.json(data);
-    }, function(error) {
-      next(error);
-    });
-  });
-
-  router.route('/verifysession').post(function(req, res, next) {
-    delegate.verifysession().then(function(data) {
-      res.json(data);
-    }, function(error) {
-      next(error);
-    });
-  });
-
-  router.route('/revokesession').post(function(req, res, next) {
-    delegate.revokesession().then(function(data) {
-      res.json(data);
-    }, function(error) {
-      next(error);
-    });
-  });
-
-  return router;
-}
-
 module.exports = function(app, guid) {
```

Comment:
@nialldonnellyfh https://github.com/feedhenry-raincatcher/raincatcher-user/blob/master/lib/router/cloud.js#L3 is not guid of service? Better rename it to serviceGuid?